lib/browser/guest-view-manager.ts

```diff
@@ -81,7 +96,7 @@ const createGuest = function (embedder, params) {
   });
 
   // Init guest web view after attached.
-  guest.once('did-attach', function (event) {
+  guest.once('did-attach', function (this: any, event: Electron.IpcRendererEvent): void {
```

Comment:
`this` is the "guest" so should be typed as `WebContents`

Reply:
Will also be adding `attachParams` and `viewInstanceId` to `WebContents` to satisfy TS.